proposals/lip-0043.md

```diff
@@ -209,64 +205,63 @@ Let `trs` be a transaction with module ID `MODULE_ID_INTEROPERABILITY` and comma
 * Let `netId` be the hash digest of `SHA-256(trs.params.genesisBlockId || senderAddress)` where `||` indicates bytes concatenation and `senderAddress` is the address of the user account corresponding to the `trs.senderPublicKey` property:
   * Then `netId` has to be unique with respect to the set of already registered sidechain network IDs in the blockchain state.
 That is the entry with store prefix equal to `STORE_PREFIX_NETWORK_ID` and store key equal to `trs.params.netID` does not exist in the store.
-* The `trs.params.initValidators.keys` array must contain at least 1 element (1 public key) and at most  `MAX_NUM_VALIDATORS` elements.
-* Every element in the `trs.params.initValidators.keys` array has to be unique.
-* Elements in the `trs.params.initValidators.keys` array have a length of 48 bytes.
-* The `trs.params.initValidators.keys` array must be ordered lexicographically.
-* The `trs.params.initValidators.weights` array must contain the same number of elements as `trs.params.initValidators.keys`.
-* Elements in the `trs.params.initValidators.weights` array are positive integers distinct from 0.
-* The range of valid values of the `trs.params.initValidators.certificateThreshold` property is given by the total sum of the validators weights:
+* Rules for `trs.params.initValidators` array:
+    *   The array must have at least 1 element and at most `MAX_NUM_VALIDATORS` elements.
+    *   The array must be ordered lexicographically by `blsKey` property.
+    *   The `blsKey` property of each element has a length of 48 bytes.
+    *   The `blsKey` property of each element is unique within the array.
+    *   The `bftWeight` property of each element is a positive integer.
+    *   Let `totalWeight` be the sum of the `bftWeight` property of every element in the `trs.params.initValidators` array. Then `totalWeight` has to be less than or equal to `MAX_UINT64`.
+* The range of valid values of the `trs.params.certificateThreshold` property is given by the total sum of the validators weights:
   * Minimum value: ⌊ ⅓  × `totalWeight` ⌋
   * Maximum value: `totalWeight`
 
-where ⌊⋅⌋ is the floor function and `totalWeight` is the total sum of every element in the `trs.params.initivalidators.weights` array.
+where ⌊⋅⌋ is the floor function.
```

Comment:
I think the bound on the certificate threshold is incorrect. It should be
```
 floor(1/3*aggregateBFTWeight)+1 <= certificateThreshold,
 ```
 see https://research.lisk.com/t/introduce-bft-module/321#properties-12. I would also just use the notation `floor()` instead of the bracket notation if you need to explain the notation every time.

Reply:
thanks for the updated, fixed and updated as suggested